common-summoning.lic

```diff
@@ -90,6 +90,46 @@ module DRCS
     return nil
   end
 
+  def check_slivers(game_state)
```

Comment:
Should this be in DRCS or DRCMM?

Reply:
It is summoning but we do have a `DRCMM`, `common-moonmage.lic` file. I believe it should go into there. Also, I assume you tested this and made sure when it calls this the `game_state` is known.

Reply:
I colocated it with moonblade, if I move it to drcmm, I will probably move more. Changing this to draft for now.